src/components/pages/SavedJudges/SavedJudges.js

```diff
@@ -18,16 +18,16 @@ function SavedJudges({ savedJudges, deleteSavedJudge }) {
       options: {
         filter: true,
       },
-      //Link to individual judge page
       renderCell: params => (
         <>
-          <Link
+          {/* <Link
             to={`/judge/${params.value.split(' ').join('%20')}`}
             className="savedJudgeLink"
-          >
-            <span>{params.value}</span>
-          </Link>
+          > */}
+          <span>{params.value}</span>
+          {/* </Link> */}
         </>
+        // ^^produces error after judge is deleted - bug
       ),
```

Comment:
I like that you added your own comment when you commented out the code so we know to keep it and look into it later.